backend/node_app/config/constants.js

```diff
@@ -43,7 +43,17 @@ module.exports = Object.freeze({
 				port: 5432,
 				dialect: 'postgres',
 				logging: false
+			},
+			eda: {
+				username: process.env.POSTGRES_USER_EDA,
```

Comment:
Make sure these EDA env vars are added to the template and we need to make sure they get into the chart values for k8s deployments.